cmd/server/server.go

```diff
@@ -18,6 +18,8 @@ import (
 	"context"
 	"crypto/tls"
 	"errors"
+	"github.com/woodpecker-ci/woodpecker/server/plugins/encrypted_secret_store"
```

Comment:
please sort imports (this should be in lower group for local imports)

Reply:
fixed